app/templates/crate/owners.hbs

```diff
@@ -57,7 +57,11 @@
         {{/link-to}}
       </div>
       <div class='stats'>
-        {{{ if user.email user.email "&nbsp;" }}}
+        {{#if user.email}}
+          {{html-safe user.email}}
```

Comment:
Does the user email really need `html-safe`? I believe the triple braces were only there to prevent `&nbsp;` from being escaped, and if you expand it the way you did we don't need it anymore.

Reply:
Hm, good point. I tried to keep the exact same behaviour as I wasn't sure, but if we don't need it here then even better!

Reply:
Thinking about it, marking the email address as "safe" even appears to be actively harmful, and we don't do so in other places we show email addresses. This currently doesn't _actually_ introduce an XSS vulnerability since the only email address we show is the address of the current user, so people could only attack themselves, but I'd prefer if you remove `html-safe` here.

Reply:
Done!